govc/flags/enum.go

```diff
@@ -0,0 +1,91 @@
+package flags
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+)
+
+// enum is a value store that validates against a set of strings. The
+// validation is performed at set-time.
+type enum struct {
```

Comment:
Nice.  We can make use of these in some of the other commands too.  At some point I'd like to ditch our custom CLI framework for something else, but not in a rush to do that.